terrautils/metadata/lemnatec.py

```diff
@@ -35,31 +43,41 @@
 SENSOR_VNIR = "VNIR"
 SENSOR_WEATHER = "weather"
 
+logging.basicConfig()
+logger = logging.getLogger("terrautils.metadata.lemnatac")
 
-def clean(metadata, sensorId):
-    """ Given a LemnaTec metadata.json object, produces the "cleaned" metadata that 
+def clean(metadata, sensorId, filepath=""):
+    """ 
+    Given a LemnaTec metadata.json object, produces the "cleaned" metadata that 
     will be put in the Clowder jsonld endpoint.
     """
     
     orig_lem_md = metadata['lemnatec_measurement_metadata']
     
     properties = {}
-    properties["gantry_variable"] = _standardize_gantry_system_variable_metadata(orig_lem_md)
-    properties["sensor_fixed"]    = _get_sensor_fixed_metadata_url(sensorId)
-    properties["sensor_variable"] = _standardize_sensor_variable_metadata(sensorId, orig_lem_md, properties["gantry_variable"])
+    properties["gantry_variable_metadata"] = _standardize_gantry_system_variable_metadata(orig_lem_md, filepath)
+    properties["gantry_fixed_metadata"]    = _get_sensor_fixed_metadata_url(PLATFORM_SCANALYZER)
+    properties["sensor_fixed_metadata"]    = _get_sensor_fixed_metadata_url(sensorId)
+    properties["sensor_variable_metadata"] = _standardize_sensor_variable_metadata(sensorId, orig_lem_md, 
+                                                    properties["gantry_variable_metadata"], filepath)
     #_standardize_user_given_metadata(orig_lem_md)
 
     return properties
             
-# Standardize user given metadata
+
 def _standardize_user_given_metadata(lem_md):
-    # Currently not used. Will possibly return reference to user given metadata in BETY
+    """
+    Currently not used. Will possibly return reference to user given metadata in BETY
```

Comment:
Good point. This was a stub while we decided what to do with the user_given metadata.  If it will always be pulled from BETYdb, it can certainly move up a level.